### PR TITLE
Added the Bozorgnia & Campbell (2016) GMPE for V/H (PR Re-open)

### PR DESCRIPTION
**Pull request by ftbernales**
Re-opened PR from my colleague @stansays via   ##gem#6935.

Cherry-picked his commits to this new branch as my origin/master (from which the previous PR is from) is having problems.

### CODE REVIEW

**Comment by ftbernales** [comment]
Checks OK. Merged latest changes from [upstream/master ](https://github.com/gem/oq-engine)especially commit 51666b1a278d183d389c754a8e37f9277c22d70b which introduced a new IMC for our V/H spectral ratio models.

File: openquake/hazardlib/gsim/bozorgnia_campbell_2016_vh.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    """
+    VGMPE = BC15.BozorgniaCampbell2016()
+    HGMPE = CB14.CampbellBozorgnia2014()
+
+    #: Supported tectonic region type is active shallow crust
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.ACTIVE_SHALLOW_CRUST
+
+    #: Supported intensity measure types are spectral acceleration, peak
+    #: ground velocity and peak ground acceleration
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
```
Comment (mmpagani) [inline openquake/hazardlib/gsim/bozorgnia_campbell_2016_vh.py:129]: better to define this as {PGA, PGV, SA}

**Review by mmpagani — APPROVED** [review]
Apart from the minor note LGTM. Good job!

**Comment by micheles** [comment]
In the changelog, should we chance [Stanley Sayson] with  [Stanley Sayson, Francis Jenner Bernales] ?

**Comment by ftbernales** [comment]
> In the changelog, should we chance [Stanley Sayson] with [Stanley Sayson, Francis Jenner Bernales] ?

@micheles [Stanley Sayson] is fine. He's the one who authored the commits :)

**Pull request merged**